.azure-pipelines/bazel.yml

```diff
@@ -20,6 +20,10 @@ parameters:
 - name: envoyBuildFilterExample
   type: string
   default: ""
+- name: cacheTestResults
+  displayName: "Cache test results"
+  type: string
+  default: "true"
```

Comment:
there are 11 types of people, those who understand binary, those who dont, and microsoft